jingo_minify/tests.py

```diff
@@ -240,7 +243,7 @@ def test_static_override():
 def test_css(getmtime, time):
     getmtime.return_value = 1
     time.return_value = 1
-    env = jingo.env
+    env = jingo.get_env()
```

Comment:
Yeah, up to you - since it was tests I figured updating only requirements would work - I don't know if jingo-minify itself is still compatible with jingo < 0.8